web/src/popover_menus.ts

```diff
@@ -332,10 +332,14 @@ export function on_show_prep(instance: tippy.Instance): void {
 function get_props_for_popover_centering(
     popover_props: Partial<tippy.Props>,
 ): Partial<tippy.Props> {
+    let placement: tippy.Placement = "top";
+    if (util.is_mobile()) {
+        placement = "bottom";
+    }
     return {
         arrow: false,
         getReferenceClientRect: () => new DOMRect(0, 0, 0, 0),
-        placement: "top",
+        placement,
```

Comment:
Can you explain how this fixes the issues? Also, this change seems to be effecting more than just emoji popover.

Reply:
where is the explanation you have put?